packages/content-api/index.js

```diff
@@ -0,0 +1,43 @@
+const request = require('superagent');
+
+const create = ({host, version, key}) => {
+    return ['posts', 'users', 'tags', 'pages'].reduce((apiObject, resourceType) => {
+        function browse(options = {}) {
+            return makeRequest(resourceType, options);
+        }
+        function read(data, options = {}) {
+            if (!data) {
+                return Promise.reject(new Error('Missing data'));
+            }
+
+            if (!data.id && !data.slug) {
+                return Promise.reject(new Error('Must include either data.id or data.slug'));
+            }
+
+            const params = Object.assign({}, data, options);
+
+            return makeRequest(resourceType, params, data.id || `slug/${data.slug}`);
+        }
+
+        return Object.assign(apiObject, {
+            [resourceType]: {
+                read,
+                browse
+            }
+        });
+    }, {});
+
+    function makeRequest(resourceType, params, id) {
+        delete params.id;
+        return request.get(`${host}/api/${version}/content/${resourceType}/${id ? id + '/' : ''}`)
```

Comment:
This assumes that the apiHost will be passed in as `http://localhost:2368/ghost` - to be flexible for a future of not having the api mounted on `/ghost` - but seeing as that's hard coded atm, shall i just hardcode it here too?